acquia_cms.profile

```diff
@@ -1,378 +0,0 @@
-  // especially via Drush.
-  $operations = $facade->getAllOperations(TRUE);
-  $batch = [
-    'operations' => $operations,
-    'finished' => 'update_site_studio_settings',
-  ];
-
-  // Set batch along with drush backend process if site is being
-  // installed via Drush, so that we can show log on the screen during
-  // site studio package import/validate.
-  if (PHP_SAPI == 'cli') {
-    batch_set($batch);
-    drush_backend_batch_process();
-  }
-  else {
-    return $batch;
-  }
-}
-
-/**
- * Install default content as part of install task.
- */
-function install_acms_import_default_content() {
-  if (\Drupal::moduleHandler()->moduleExists('acquia_cms_image')) {
-    \Drupal::service('default_content.importer')->importContent('acquia_cms_image');
-  }
-}
-
-/**
- * Implements hook_modules_installed().
- */
-function acquia_cms_modules_installed(array $modules) : void {
-  // Don't do anything during site installation, since that can break things in
-  // a big way if modules are being installed due to changes made on the site
-  // configuration form.
-  if (InstallerKernel::installationAttempted()) {
-    return;
-  }
-
-  $module_handler = Drupal::moduleHandler();
-
-  if ($module_handler->moduleExists('acquia_telemetry')) {
-    Drupal::classResolver(TelemetryFacade::class)->modulesInstalled($modules);
-  }
-
-  if ($module_handler->moduleExists('cohesion_sync')) {
-    $module_handler->invoke('cohesion_sync', 'modules_installed', [$modules]);
-  }
-}
-
-/**
- * Implements hook_modules_uninstalled().
- */
-function acquia_cms_modules_uninstalled(array $modules) {
-  if (\Drupal::service('module_handler')->moduleExists('acquia_telemetry')) {
-    Drupal::classResolver(TelemetryFacade::class)->modulesUninstalled($modules);
-  }
-}
-
-/**
- * Method that calls another method to capture the installation end time.
- */
-function install_acms_finished() {
-  $telemetry = Drupal::classResolver(AcquiaTelemetry::class);
-  $telemetry->setTime('install_end_time');
-}
-
-/**
- * Installs additional required modules, depending on the environment.
- */
-function install_acms_additional_modules() {
-  // Call ToggleModules Service.
-  \Drupal::service('acquia_cms_common.toggle_modules')->ToggleModules();
-  // Save configuration for imagemagick.
-  if (Environment::isAhEnv() && !Environment::isAhIdeEnv()) {
-    $moduleHandler = \Drupal::service('module_handler');
-    if ($moduleHandler->moduleExists('imagemagick')) {
-      \Drupal::configFactory()
-        ->getEditable('imagemagick.settings')
-        ->set('path_to_binaries', '/usr/bin/')
-        ->save();
-      \Drupal::configFactory()
-        ->getEditable('system.image')
-        ->set('toolkit', 'imagemagick')
-        ->save();
-    }
-  }
-
-  /*
-   * We've to explicitly set purge plugin as acquia_purge, If Acquia Purge
-   * module is enabled, else this would give below error:
-   *
-   * ERROR: Purgers:There is no purger loaded which means that you need a module
```

Comment:
@rajeshreeputra We've removed this now. I think this error was coming when installing site using existing config. I think @chandan-singh7929 has created one ticket to validate installing site using existing config. Please include this as well in that ticket to make sure, we test this and don't get any error.
cc: @chandan-singh7929

Reply:
Yes, this is included in the ticket